TaskWithMaxMinSortForArray/TaskWithMaxMinSortForArray.cpp

```diff
@@ -0,0 +1,59 @@
+﻿#include <iostream>
+
+int searchMax(int array[], size_t len) {
+	int max = array[0];
+	for (int i = 0; i < len; i++)
```

Comment:
Maybe int i = 1 enough? The same for searchMin.

Reply:
Fixed